frontend/src/entities/entities.js

```diff
@@ -0,0 +1,21 @@
+
+export const Region = {
```

Comment:
Remind me to have somebody educate you on how to COMMENT YOUR F#$KING CODE.

Reply:
switching to typescript, don't need that many comments for such simple/self-explained objects... calm down man. Added simple comments.